src/wasm-lib/kcl/src/parser/parser_impl.rs

```diff
@@ -2237,6 +2257,34 @@ fn fn_call(i: TokenSlice) -> PResult<Node<CallExpression>> {
     })
 }
 
+fn fn_call_kw(i: TokenSlice) -> PResult<Node<CallExpressionKw>> {
+    let fn_name = identifier(i)?;
+    opt(whitespace).parse_next(i)?;
+    let _ = open_paren.parse_next(i)?;
+    ignore_whitespace(i);
+
+    let initial_unlabeled_arg = opt((expression, comma, opt(whitespace)).map(|(arg, _, _)| arg)).parse_next(i)?;
+    let args = labeled_arguments(i)?;
+    if let Some(std_fn) = crate::std::get_stdlib_fn(&fn_name.name) {
+        let just_args: Vec<_> = args.iter().map(|la| &la.arg).collect();
```

Comment:
I don't think this is right since it requires the args to be in the correct order, but we should support any order

Reply:
I'm not going to do the typechecking here yet. I think we'll need to modify the derive_docs::stdlib proc-macro to get this right.